libindy/build.rs

```diff
@@ -32,4 +32,38 @@ fn main() {
             }
         }
     }
+		match &target {
```

Comment:
please clean-up code here. Seems like this section should be in `else if` for previous one (windows)